dns.go

```diff
@@ -126,9 +130,17 @@ func probeDNS(target string, w http.ResponseWriter, module Module, registry *pro
 		fallbackProtocol = "ip6"
 	}
 
+	targetAddress, targetPort, _ = net.SplitHostPort(target)
+	if targetPort == "" {
+		// missingPort could be fixed by fallbacking to the default port.
```

Comment:
I think it should function the same way, as we previously didn't check the error return at all, and would simply send the empty string target returned by `net.SplitHostPort()` to the DNS lookup.  At this point we would catch the error.